### ISSUE TITLE
api calls are failing with 'forbidden'

### ISSUE THREAD

**Issue by khassel** [derhuerst/flightradar24-client#29]
I'm not sure if this project is still maintained but since a few days it doesn't work anymore.

Api calls using `radar` come back with `forbidden` error.

I think they put your `User-Agent` header on a black list, when changing this to another value it works again ...

**Comment by jurasjo** [comment]
Yes, I can confirm. After replacing User-Agent to the generic Firefox one, it works. At least for now.
```const headers = {
	//'User-Agent': 'https://github.com/derhuerst/fetch-flightradar24-flights'
	'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:108.0) Gecko/20100101 Firefox/108.0'
}
```

**Comment by UnclePhil** [comment]
Little hack !!
On my side, i change the code  with this rotation of user-agent code to avoid the flooding symptom (and bot reaction) on the FR server.


```
// const headers = {
// 	'User-Agent': 'https://github.com/derhuerst/fetch-flightradar24-flights'
// }

/// PKO header

const uas= ("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.246"
,"Mozilla/5.0 (X11; CrOS x86_64 8172.45.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.64 Safari/537.36"
,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/601.3.9 (KHTML, like Gecko) Version/9.0.2 Safari/601.3.9"
,"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.111 Safari/537.36"
,"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:15.0) Gecko/20100101 Firefox/15.0.1"
,"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36 Edg/114.0.1823.51"
,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.5.1 Safari/605.1.15"
,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.3"
,"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.6,2 Safari/605.1.1"
,"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 OPR/99.0.0.") 

const headers = {
	'User-Agent': uas[Math.floor(Math.random()*uas.length)]
}
```

**Comment by FatPilot** [comment]
The patch also worked for me!
Thank you for sorting this out!
I'm really very impressed with this community and it's response.

**Comment by derhuerst** [comment]
Fixed as of 5e96538. Published as `flightradar24-client@0.3.1`. 🚀
